### ISSUE TITLE
'follow-me' hovers

### ISSUE THREAD

**Issue by nicolaskruchten** [plotly/plotly.js#3899]
A new hover mode variant that would cause hoverlabels to follow the mouse rather than be pinned to a specific location on the plot. e.g. https://echarts.baidu.com/echarts2/doc/example/bar1.html#-en or https://www.fusioncharts.com/ (scroll down for demo)

**Comment by nicolaskruchten** [comment]
https://www.datawrapper.de/ does it also (see their big bubble chart)

**Comment by alexcjohnson** [comment]
God I hate mouse-following so much... the hover label is not labeling the mouse pointer, it's labeling the data point! But yes, if we also add hover *styling* so the point being described somehow pops out, I guess in principle it could be allowable.

**Comment by nicolaskruchten** [comment]
Yeah the way Datawrapper does it is pretty nice with the hover-styling.

**Comment by etpinard** [comment]
Duplicate of https://github.com/plotly/plotly.js/issues/2696 ?

**Comment by nicolaskruchten** [comment]
not sure about the relationship to #2696

**Comment by etpinard** [comment]
Ah I see, this ticket isn't about adding hover label (and events) to spaces in-between data points, but to move the labels on a given picked datum. My bad.

**Comment by pawchen** [comment]
I have trouble with hover labels, they are blocking the lines under it. Currently I'm setting transparent background color to workaround. It would be great to have a solution for the label to avoid the lines. Allowing the label to following the cursor sticking alongside the spike could be a good option.

![image](https://user-images.githubusercontent.com/159110/166730266-3a313e86-3459-417d-bd2f-5b50fe63372b.png)
